solutions/cfgym-102951C.mdx

```diff
@@ -47,16 +47,18 @@ vector<int> lis;
 
 int main() {
 	cin.tie(0)->sync_with_stdio(0);
-
 	cin >> n;
+	
```

Comment:
Why add these newlines? Doesn't improve readability.

Reply:
Hi, as far as I know I haven't made any changes to the C++ Code of this solution. I've only contributed to the Java Solution.
Do you think that I may have added in those newlines by mistake because I don't remember making any changes to the C++ Code.

Reply:
Yeah, doesn't look like you were the one who added them. Could you remove them nonetheless?

Reply:
Okay, I've removed them. Lemme know if I've done something wrong or If I need to change something else.